lib/OpenQA/Worker/Jobs.pm

```diff
@@ -632,6 +632,8 @@ sub start_job {
         log_warning('job errored. Releasing job', channels => ['worker', 'autoinst'], default => 1);
         return stop_job("job run failure");
     }
+    my $isotovideo_pid = $worker->{child}->pid() // 'unknown';
```

Comment:
Given that we know that the child was started successfully at this point, can it be unset? Should this be a failure condition instead of `unknown`?

Reply:
This value being `undef` lead to the warning which caused the tests to fail:

```
#   Failed test 'no (unexpected) warnings (via done_testing)'
#   at ./t/10-jobs.t line 710.
```

Maybe the error handling should be improved here. But so far I only want to add some further logging and not change the logic.

Reply:
Thanks for clarifying. I couldn't see it in the history.